README.md

```diff
@@ -1,32 +1,58 @@
-# bridge-ui
+# Bridge-ui
 
-Empty project.
+This repo hosts a simple interface to send tokens using the [Rialto bridge](https://github.com/paritytech/parity-bridges-common) between an Ethereum network and a Substrate network.
+Metamask is required to sign Ethereum transactions.
 
-## Building and running on localhost
+The interface is available at: https://paritytech.github.io/bridge-ui/
```

Comment:
Maybe make it a link as well?

Reply:
On Github it's changed into a link automatically actually. Making it explicit though.
```suggestion
The interface is available at: [https://paritytech.github.io/bridge-ui](https://paritytech.github.io/bridge-ui)
```